.github/workflows/semanticore.yml

```diff
@@ -0,0 +1,25 @@
+name: Semanticore
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  semanticore:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.*' ]
```

Comment:
I guess we need no matrix for only one entry. Please use the go version directly in the setup job